schema-admin.graphql

```diff
@@ -358,7 +355,7 @@ type ScheduledCorpusItem {
     """
     The associated Approved Item.
     """
-    approvedItem: ApprovedCorpusItem!
+    approvedItem: ApprovedCorpusItem
```

Comment:
I have made this optional to reuse this type in the return value of `scheduledSurfaceHistory` subquery. Open to suggestions here! Originally I thought we should set up a very abridged custom type that would only show date + surface guid, but then (see above notes on future-proofing it) decided to return the whole thing. 

But not the approved item! We already have it in the main query and it doesn't make sense to pull it in again.

Reply:
I don't recommend making this nullable just based off the parent context. Better to use a custom type, return it as expected (and put in reasonable query depth limits to prevent weird circular resolvers), and use dataloaders to cache.

Reply:
Agreed @kschelonka we should keep this required given the context and put in query depth limits